orquesta/events.py

```diff
@@ -98,6 +102,7 @@
 TASK_PAUSED_WORKFLOW_ACTIVE = 'task_paused_workflow_active'
 TASK_PAUSED_WORKFLOW_DORMANT = 'task_paused_workflow_dormant'
 TASK_RESUMING = 'task_resuming'
+TASK_RERUNNING = 'task_rerunning'
```

Comment:
Remove if these events are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.